deck-helper/RestrictedList.js

```diff
@@ -14,6 +14,15 @@ class RestrictedList {
                   banned: this.rules.banned,
                   pods: this.rules.pods
               };
+
+        if (!formatRules) {
+            return {
+                name: format === 'tower of joy' ? 'Tower of Joy' : format,
+                valid: true,
+                version: ''
+            };
+        }
```

Comment:
When changing the format to `draft`, I'd just update this to the following (fine with not considering VDS for now)

```suggestion
        if (!formatRules && format === 'draft') {
            return {
                name: 'Tower of Joy',
                valid: true,
                version: ''
            };
        }
```

Notably, there is "draft" rules checking further down but it will need to be revisited/considered once draft is actually implemented on TiT (as it realistically should be locking decks to a draft cube event)